samples/bluetooth/central_smp_client/README.rst

```diff
@@ -1,13 +1,13 @@
 .. _bluetooth_central_dfu_smp:
 
-Bluetooth: Central DFU SMP
-##########################
+Bluetooth: Central SMP Client
+#############################
 
 .. contents::
    :local:
    :depth: 2
 
-The Central DFU SMP sample demonstrates how to use the :ref:`dfu_smp_readme` to connect to an SMP Server and send a simple echo command.
```

Comment:
`dfu_smp_readme` should also be updated then?

Reply:
I think not, because this is a link to the service description - GATT DFU SMP Service Client.